libs/ipc_connector/tests/ipc_manager_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <boost/asio.hpp>
+#include <boost/ut.hpp>
+#include <glaze/glaze.hpp>
+#include <tfc/confman/file_storage.hpp>
+#include <tfc/ipc_connector/dbus_server_iface.hpp>
+#include <tfc/progbase.hpp>
+
+template <typename storage_t>
+struct file_storage_mock {
```

Comment:
this seems nice, we should later make mock classes more exposed so it does not need to be redeclared